app/models/account.rb

```diff
@@ -120,7 +120,7 @@ def groups
   end
 
   def add_group(owner)
-    with_valid_account_type('user') { set_parent(owner) }
+    with_valid_account_type('user') { add_parent(owner) }
```

Comment:
Doh, I just read [the comment above](https://github.com/ManageIQ/manageiq/pull/14060#issuecomment-282422063).  So anyway, it makes sense, but to your point, there is probably untested code that might need to be changed.  It's such a hard regex to find bad uses of `parent=`.  😢